.github/dependabot.yml

```diff
@@ -0,0 +1,20 @@
+version: 2
+updates:
+  - package-ecosystem: npm
+    directory: "/"
+    schedule:
+      # Going to start with a high interval, and then tone it back
+      interval: daily
```

Comment:
I thought for now the interval would be 'monthly' 🤷‍♀️

Reply:
I'd like to see more PRs at first to catch up on outdated packages.